docs/tutorial/tasks.md

```diff
@@ -73,14 +61,19 @@ Running the flow in your terminal will result in something like this:
 ```
 </div>
 
-And you should now see this task run tracked in the UI as well.
+You should also be able to see this task and its dependencies in the flow run graph in the UI.
 
-## Caching
+## Reuse tasking results
 
-Tasks support the ability to cache their return value.
-Caching allows you to efficiently reuse [results](/concepts/results/) of tasks that may be expensive to reproduce with every flow run, or reuse cached results if the inputs to a task have not changed.
+Using tasks, you can efficiently reuse task results that are expensive to reproduce with every flow run or if the inputs to a task remain unchanged.
 
-To enable caching, specify a `cache_key_fn` — a function that returns a cache key — on your task. You may optionally provide a `cache_expiration` timedelta indicating when the cache expires. You can define a task that is cached based on its inputs by using the Prefect [`task_input_hash`](/api-ref/prefect/tasks/#prefect.tasks.task_input_hash). Let's add caching to our `get_url` task:
+To enable caching:
+
```

Comment:
I like the switch to a list format here to make things clearer. 

Using - instead of * for consistency with most list formats we use.

I think some of the meaning was original meaning was lost in the new bullets points, so tried to clarify.